### DISCUSSION TITLE
Can we run this deployment without FreeIPA?

### DISCUSSION THREAD

**Discussion by parimalpatil28** [cloudera-labs/cloudera-deploy#146] [Help]
Hello @wmudge,
I need  help, I am working on one project in which I want to create a CDP PVC BASE 7.1.x cluster from scratch using jenkins and devops tools and with the help of this cloudera deployed project.
So, can we skip the FreeIPA step or may be replace it?
As my security teams won't allow to use that thing

Thanks in advance,
Parimal

**Comment by wmudge** [comment]
You can definitely replace it; FreeIPA is not a hard dependency.  The underlying roles in `cloudera.cluster` that `cloudera-deploy` uses can set up a CA server and MIT KDC. Moreover, you can specify existing resources via the config parameters. (For example, we do this several internal clusters that rely on MSFT AD for LDAP and KDC.)
